sled-agent/src/sim/server.rs

```diff
@@ -450,7 +489,16 @@ pub async fn run_standalone_server(
         None => vec![],
     };
 
+    let services =
```

Comment:
If I'm understanding right, we're still constructing `Service`s for these zones and still inserting those in the db.  Is that right?  (I think that's the right call until we finish #4947.)

Reply:
That's right. I think there are three steps:

1. Ensure new systems start off with a blueprint set (this PR)
2. Ensure existing systems get a blueprint set (support operation)
3. Remove the `service` table and all the fallout from int (including dropping `Service`s from the RSS handoff) will be a followup PR that can be in flight or even landed concurrently with 2, as long as we do 2 before deploying 3